RAG-191 Create Walmart SDK

shivangeegodhani opened the pull request:
Change version in pom file.
Add annotation in Access Credential class.
Create new Builder Method in WalmartSDK.

shivangeegodhani committed:
RAG-191
Change version in pom file.
Add annotation in AccessCredential class.
Create new Builder Method in WalmartSDK.

dftrakesh commented on src/main/java/io/github/dft/walmartsdk/WalmartSDK.java:
```diff
@@ -44,6 +44,14 @@ public WalmartSDK(AccessCredential accessCredential) {
         this.accessCredential = accessCredential;
     }
 
+    public ItemsAPI getItemsAPI(){
```
Remove it from SDK class. This Class should be used independently

shivangeegodhani replied:
Ok sir.

dftrakesh commented on src/main/java/io/github/dft/walmartsdk/WalmartSDK.java:
```diff
@@ -44,6 +44,14 @@ public WalmartSDK(AccessCredential accessCredential) {
+        return new ItemsAPI(accessCredential);
+    }
+
+    public OrdersAPI getOrdersAPI(){
```
Remove it from SDK class. This Class should be used independently

shivangeegodhani replied:
Ok sir.

shivangeegodhani committed:
RAG-191
Change version in pom file.
Remove ItemAPI and OrderAPI class method from WalmartSDK class.
